### ISSUE TITLE
[ESC] HAKRC 35A

### ISSUE THREAD

**Issue by cctsao1008** [AlkaMotors/AM32_Arterytek#3]
![image](https://user-images.githubusercontent.com/543608/227679791-e68a4e9f-25bd-4447-a051-1267eefd90ce.png)

Does this build support the HAKRC 35A that uses the AT32F421 MCU from Artery?

Thank you.

BR CCTSAO

**Comment by AlkaMotors** [comment]
Yes, it should work with that esc. We would need to know some more details about it. like which mcu it uses, the 4x4mm one or the 5x5mm one?

**Comment by cctsao1008** [comment]
okay, I will check it and feedback to you, thank you.

BR CCTSAO

**Comment by cctsao1008** [comment]
> Yes, it should work with that esc. We would need to know some more details about it. like which mcu it uses, the 4x4mm one or the 5x5mm one?

![IMG20230401235142](https://user-images.githubusercontent.com/543608/229301338-624d2d36-d334-4114-a0c4-a8226b7752da.jpg)
![IMG_20230401_235838](https://user-images.githubusercontent.com/543608/229301345-3aef3d9a-4dee-437d-99bb-5c776a4d86cb.jpg)

**Comment by AlkaMotors** [comment]
That one should use the hackRc_k target and the pb4 bootloader.
https://github.com/AlkaMotors/AM32-MultiRotor-ESC-firmware/releases/download/v1.95/AM32_HAKRC_K_F421_1.95.hex

**Comment by cctsao1008** [comment]
Hello @AlkaMotors  :

Thank you for the reply, and I would like to confirm with you that the bootloader version you mentioned is the following highlighted one.

![image](https://user-images.githubusercontent.com/543608/230875325-4d8ebdcf-eef1-4c18-a7ed-354b9fe9e0a8.png)

**Comment by cctsao1008** [comment]
Hello @AlkaMotors :

It works!

![image](https://user-images.githubusercontent.com/543608/232326399-7d083f46-270d-4897-9da9-3d5cb1d4efc2.png)

I using the Segger Jlink to flash the bootloader with the following additional software tools

![image](https://user-images.githubusercontent.com/543608/232326511-a81ad2fc-f0e7-48d6-bec2-865fe7d11e2b.png)


BR CCTSAO

**Comment by Fahmi1212** [comment]
can u tell me what the target firmware the esc AM32, i have same MCU AT32f421,

**Comment by Fahmi1212** [comment]
Can I Flash With ST-Link V2?
